addons/escoria-core/game/escoria.gd

```diff
@@ -388,3 +388,25 @@ func _on_settings_loaded(p_settings: ESCSaveSettings) -> void:
 	)
 	TranslationServer.set_locale(settings.text_lang)
 
+
+# Input function to manage specific input keys
+func _input(event):
+	if event.is_action_pressed("esc_show_debug_prompt"):
+		escoria.main.get_node("layers/debug_layer/esc_prompt_popup").popup()
+	
+	if event.is_action_pressed("ui_cancel"):
+		inputs_manager._on_pause_menu_requested()
```

Comment:
Please emit a signal and connect the _on_pause_menu_requested to it in inputs_manager instead of calling a private method

Reply:
Done